code/modules/cargo/packs.dm

```diff
@@ -1810,6 +1820,7 @@
 
 /datum/supply_pack/misc/janitor/lightbulbs
 	name = "Replacement Lights"
+	desc = "May the light of Aether shine upon this station! Or at least, the light of forty two light tubes and twenty one light bulbs."
```

Comment:
is this a metroid prime 2 reference if so i love you

Reply:
yeaaah buddy